app/javascript/components/layout/Header.js

```diff
@@ -7,6 +7,10 @@ class Header extends React.Component {
     super(props);
   }
 
+  getUserRoleIcon = role => {
```

Comment:
This function isn't used anywhere and doesn't actually do anything.

Reply:
removed!